packages/util-use-case/lib/adapters/http/index.mjs

```diff
@@ -0,0 +1,16 @@
+// TODO: gh-55 move to http adapters, introduce runner adapter concept
+
+export function toUseCaseParams() {
+  return {}
+}
+
+export function toUseCaseContext() {
+  return {}
+}
+
+export function mapResultToHttp(result, req, res) {
+  res.send(result)
+}
+
```

Comment:
not sure about the project but in production, I use a stream piping to response. also need to think about configuring of the response headers specific to a endpoint

Reply:
Good point. Could you drop a few samples with pipe procession for the response? 

> configuring of the response headers specific to an endpoint

For example ?

Reply:
I made something like this:
```function mapToResArchiveCsv({ archive, nextPage }, res, req) {
    archive.pipe(res);
    if ((req.query &&  (req.query.format || 'json').toLowerCase()) === 'csv') {
        res.writeHead(200, {
            'Content-Type': 'application/csv',
            'Content-disposition': 'attachment; filename=position_history.csv',
            'Content-Next-Positions-Page': nextPage,
        });
    } else {
        res.writeHead(200, {
            'Content-Type': 'application/zip',
            'Content-disposition': 'attachment; filename=position_history.zip',
            'Content-Next-Positions-Page': nextPage,
        });
        archive.finalize();
    }
}

module.exports = {
    get: makeRequestHandler(PositionGet,
        (req) => ({
            project_uid: req.get('X-User-Project'),
            page: req.query.page,
            sublocation_uid: req.query && (req.query.sublocation_uid || req.query.area_uid),
            trackable_objects: req.query && (req.query.trackable_objects || req.query.track_uid),
            from: req.query && req.query.from,
            to: req.query && req.query.to,
            format: req.query &&  (req.query.format || 'json').toLowerCase(),
        }),
        (result, res, req) => {
            mapToResArchiveCsv(result, res, req);
        })
};```

Reply:
extracted to #90